src/features/form/validations.jsx

```diff
@@ -59,6 +72,26 @@ const minDateValue = (minDate, reference, customText) => value => {
   }
 };
 
+const generateAllowDecimalsValidator = concept => {
+  const {
+    allowDecimal,
+    datatype,
+  } = concept;
+  let allowDecimalValidation;
+
+  if (typeof datatype !== 'undefined'
+    && datatype !== null
+    && typeof datatype.name !== 'undefined'
+    && datatype.name === 'Numeric'
+    && allowDecimal === false) {
+
+    allowDecimalValidation = allowDecimals();
+    return [allowDecimalValidation].filter(Boolean);
```

Comment:
I don't think you need the .filter(Boolean) here, just return [allowDecimalValidaton]

I didn't quite understand at first what that was doing in the other generate functions, but I believe it just allowed a easy way to filter out undefined validation functions, which you don't need to do here because you will only get here if allowDecimalValidation is defined.